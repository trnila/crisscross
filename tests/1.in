4 4
ahoj
h  o
o  h
joha