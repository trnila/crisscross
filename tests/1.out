┌─┬─┬─┬─┐
│a│h│o│j│
├─┼─┴─┼─┤
│h│   │o│
├─┤   ├─┤
│o│   │h│
├─┼─┬─┼─┤
│j│o│h│a│
└─┴─┴─┴─┘

